libs/eavmlib/src/esp.erl

```diff
@@ -176,7 +197,7 @@ nvs_get_binary(Namespace, Key, Default) when
     end.
 
 %%-----------------------------------------------------------------------------
-%% @doc Equivalent to nvs_set_binary(?ATOMVM_NVS_NS, Key, Value).
+%% @doc (Deprecated) Equivalent to nvs_set_binary(?ATOMVM_NVS_NS, Key, Value).
```

Comment:
Instead of a comment, you can use xref's -deprecated attribute.

https://www.erlang.org/doc/man/xref.html

Reply:
Also I think edoc doesn't use the attribute and needs its own `@deprecated` tag.

Reply:
Fixed.